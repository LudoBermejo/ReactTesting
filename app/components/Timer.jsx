import React from 'react';
import Clock from 'Clock';

const Timer = () => (
  <div>
    <p>Timer component</p>
    <Clock />
  </div>
);

module.exports = Timer;
