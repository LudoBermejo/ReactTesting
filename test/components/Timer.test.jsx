import React from 'react';
import renderer from 'react-test-renderer';

import Timer from 'Timer';

describe('Timer component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Timer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});