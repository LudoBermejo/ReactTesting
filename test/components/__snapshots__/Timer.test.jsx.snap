exports[`Timer component renders correctly 1`] = `
<div>
  <p>
    Timer component
  </p>
  <div>
    This is my clock
  </div>
</div>
`;
